integration_test/agents/agents.go

```diff
@@ -596,14 +596,6 @@ func isRetriableInstallError(platform string, err error) bool {
 		strings.Contains(err.Error(), "context deadline exceeded") {
 		return true // See b/197127877 for history.
 	}
-	if platform == "rhel-8-1-sap-ha" &&
-		strings.Contains(err.Error(), "Could not refresh the google-cloud-ops-agent yum repositories") {
-		return true // See b/174039270 for history.
-	}
-	if platform == "rhel-8-1-sap-ha" &&
-		strings.Contains(err.Error(), "Failed to download metadata for repo 'rhui-rhel-8-") {
-		return true // This happens when the RHEL servers are down. See b/189950957.
-	}
```

Comment:
I wonder whether we want to keep this code for other `rhel-8-*-sap-ha` images, as the RHUI failures are still a thing…

Reply:
Added it back for `rhel-8-*-sap-ha`.